<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>299</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_load_picture">
       <property name="text">
        <string>load picture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_find_max_color">
       <property name="text">
        <string>find max color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_redraw_picture">
       <property name="text">
        <string>redraw picture</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>MAX</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="sb_R_min"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>MIN</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sb_R_max"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="sb_G_max"/>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="sb_G_min"/>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="sb_B_min"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="sb_B_max"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btn_show_only_B">
         <property name="text">
          <string>show_only_B</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btn_show_only_G">
         <property name="text">
          <string>show_only_G</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn_show_only_R">
         <property name="text">
          <string>show_only_R</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Show_HSV_color" name="hsv_widget" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="orig_picture"/>
   </item>
   <item>
    <widget class="QLabel" name="new_picture"/>
   </item>
   <item>
    <widget class="GrapherBox" name="grapher_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GrapherBox</class>
   <extends>QWidget</extends>
   <header>grapherbox.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Show_HSV_color</class>
   <extends>QWidget</extends>
   <header>show_hsv_color.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btn_load_picture</tabstop>
  <tabstop>btn_find_max_color</tabstop>
  <tabstop>btn_redraw_picture</tabstop>
  <tabstop>sb_R_min</tabstop>
  <tabstop>sb_R_max</tabstop>
  <tabstop>btn_show_only_R</tabstop>
  <tabstop>sb_G_min</tabstop>
  <tabstop>sb_G_max</tabstop>
  <tabstop>btn_show_only_G</tabstop>
  <tabstop>sb_B_min</tabstop>
  <tabstop>sb_B_max</tabstop>
  <tabstop>btn_show_only_B</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
